lib/DDG/Spice/TwitchStreams.pm

```diff
@@ -0,0 +1,25 @@
+package DDG::Spice::TwitchStreams;
+use strict;
+use DDG::Spice;
+
+name "TwitchStreams";
+source "http://www.twitch.tv";
+description "Get Twitch Stream info";
+primary_example_queries "twitch streams dota 2";
+category "special";
+topics "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/TwitchStreams/TwitchStreams.pm";
+attribution github => ["https://github.com/dwaligon", "Jason O'Donnell"];
+
+triggers startend => 'twitch streams';
```

Comment:
What about "twitch.tv" "twitchtv" or maybe even just "twitch"? (just make sure it doesn't over trigger)